spec/lib/generators/migration_generator_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'spec_helper'
+require 'rails/generators/test_case'
+require 'generators/migration/migration_generator'
+
+describe ManageIQ::Schema::MigrationGenerator do
+  include Rails::Generators::Testing::Behaviour
+  include Rails::Generators::Testing::SetupAndTeardown
+  include Rails::Generators::Testing::Assertions
+  include FileUtils
+
+  tests       ManageIQ::Schema::MigrationGenerator
+  destination Rails.root.join('tmp/generators')
+  
+  before do 
+    prepare_destination
+    allow(described_class).to receive(:miq_schema_root).and_return(destination_root)
```

Comment:
_Technically_ this is returning a Pathname where the original method didn't, which makes them unmatched.  However, if you change it as I suggested in https://github.com/ManageIQ/manageiq-schema/pull/592/files#r678321714, then they'd both be Pathnames.

Reply:
Done.